schema/deploy/computed_columns/form_change_payment_percentage.sql

```diff
@@ -9,16 +9,28 @@ create or replace function cif.form_change_payment_percentage(fc cif.form_change
 returns numeric as
 $fn$
 
-  select
-    round(
-      100 - ((-1.5) *
-      coalesce(
-        (fc.new_form_data->>'adjustedEmissionsIntensityPerformance')::numeric,
-        (cif.form_change_calculated_ei_performance(fc))
-      )
-    + 145),
-    2
-    );
+-- We need a case here becase greatest() and least() will ignore null values, while we care
+-- about returning null if both values are null.
+select case when ((fc.new_form_data->>'adjustedEmissionsIntensityPerformance')::numeric is null
+and cif.form_change_calculated_ei_performance(fc) is null)
+  then null
+  else
+    greatest(
+      least(
+        round(
+          100 - ((-1.5) *
+          coalesce(
+            (fc.new_form_data->>'adjustedEmissionsIntensityPerformance')::numeric,
```

Comment:
I just learned this from Pierre, if a value is coming from an argument, it's clearer to use $1/$2/etc. instead of an alias to avoid confusion with any aliases in the function. (I know the alias was in there before this PR, but since it's already a rework it's a convenient time to refactor)